### PR TITLE
Merge Request

### PR DESCRIPTION
**Pull request by JanardhanBS-SyncByte**
MOSIP-14483, MOSIP-14484, MOSIP-14485 all the above issue have been
addressed and committed

### CODE REVIEW

File: MockMDS/Biometric Devices/Finger/Slap/DeviceDiscovery.json [hunk 1]
```diff
@@ -1,20 +1,16 @@
...
+   	"deviceSubId":[
+   		"1", "2", "3"
+	],
+   	"callbackId":"",
+   	"digitalId":"",
+  	"deviceCode": "b692b595-3523-slap-99fc-bd76e35f190f",
+   	"specVersion":[
+   		"0.9.5"
+	],
+   	"purpose":""
```
Comment (gsasikumar) [inline MockMDS/Biometric Devices/Finger/Slap/DeviceDiscovery.json:15]: Same here as well

File: MockMDS/Biometric Devices/Iris/Double/DeviceDiscovery.json [hunk 2]
```diff
@@ -1,20 +1,16 @@
...
+   	"deviceSubId":[
+   		"1", "2", "3"
+	],
+   	"callbackId":"",
+   	"digitalId":"",
+  	"deviceCode": "b692b595-3523-iris-99fc-bd76e35f190f",
+   	"specVersion":[
+   		"0.9.5"
+	],
+   	"purpose":""
```
Comment (gsasikumar) [inline MockMDS/Biometric Devices/Iris/Double/DeviceDiscovery.json:15]: Same here as well

**Review by gsasikumar — CHANGES_REQUESTED** [review]

File: MockMDS/Biometric Devices/Face/DeviceDiscovery.json [hunk 3]
```diff
@@ -1,20 +1,16 @@
...
+   	"deviceSubId":[
+   		"0"
+	],
+   	"callbackId":"",
+   	"digitalId":"",
+  	"deviceCode": "b692b595-3523-face-99fc-bd76e35f190f",
+   	"specVersion":[
+   		"0.9.5"
+	],
+   	"purpose":""
```
[the remark this answers is not in the document]
Comment (JanardhanBS-SyncByte) [reply MockMDS/Biometric Devices/Face/DeviceDiscovery.json]: done

**Comment by JanardhanBS-SyncByte** [comment]
done

**Review by gsasikumar — APPROVED** [review]

**Pull request merged**